api/lib/domain/models/Challenge.js

```diff
@@ -278,6 +290,12 @@ export class Challenge {
     return cloneSource.get(clonedChallenge);
   }
 
+  isDeclinaisonOf(otherChallenge) {
```

Comment:
```suggestion
  isSameVersionOf(otherChallenge) {
```
J'ai un peu du mal à bien comprendre ce getter juste en le lisant. 
Pour moi une déclinaison est directement lier au proto. 
J'ai l'impression que cette méthode vérifie plutôt que l'on est dans la même version d'épreuve.